javascript/javascript2/week2/hyfBay.js

```diff
@@ -0,0 +1,60 @@
+  
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
+console.log(productsUl);
+
+function renderProducts(products) {
+    products.forEach(product => {
+        const li = document.createElement('li');
+
+        let shipsToHTML = '';
+        product.shipsTo.forEach(country => shipsToHTML += `<li>${country}</li>`);
+
+        li.innerHTML = `
+            <ul>
+                <li>${product.name}</li>
+                <li>${product.price}</li>
+                <li>${product.rating}</li>
+                <ul class="ships-to">${shipsToHTML}</ul>
+            </ul>
+        `;
+        productsUl.appendChild(li);
+    });
+}
+
+
+const searchBar = document.getElementById('searchBar');
+
+searchBar.addEventListener('keydown', function () {
+    const arr = products.filter(elm => {
+        const arrNames = elm.name.toLowerCase();
+        return arrNames.includes(searchBar.value);
```

Comment:
I suggest use `toLowerCase()` method on `searchBar.value ` to make the search request case insensitive